tilings/strategies/verification.py

```diff
@@ -32,10 +38,33 @@
 ]
 
 
-BasicVerificationStrategy = AtomStrategy
 TileScopeVerificationStrategy = VerificationStrategy[Tiling, GriddedPerm]
 
 
+class BasicVerificationStrategy(AtomStrategy):
```

Comment:
It feels like something that belongs to css instead of tilings.

Reply:
I wasn't sure how to do that in particular as assumptions are native to Tilings. Or is your point that the expected dictionary should be a method on CombinatorialClass ?